lib/ui/platform_dispatcher.dart

```diff
@@ -1158,6 +1168,22 @@ class PlatformDispatcher {
     _onSemanticsActionZone = Zone.current;
   }
 
+  /// A callback that is invoked whenever the user requests an action to be
+  /// performed on a semantics node.
+  ///
+  /// This callback is used when the user expresses the action they wish to
+  /// perform based on the semantics node supplied by updateSemantics.
+  ///
+  /// The framework invokes this callback in the same zone in which the
+  /// callback was set.
+  SemanticsActionEventCallback? get onSemanticsActionEvent => _onSemanticsActionEvent;
+  SemanticsActionEventCallback? _onSemanticsActionEvent;
+  Zone _onSemanticsActionEventZone = Zone.root;
+  set onSemanticsActionEvent(SemanticsActionEventCallback? callback) {
```

Comment:
Should we make it an error to set both `onSemanticsActionEvent` and `onSemanticsAction`?

Reply:
I opted for both are getting called if you set both. Do you feel strongly about only allowing one at a time?

Reply:
Gotcha, calling both sounds good! Marking as resolved